lib/dal/tests/integration_test/action.rs

```diff
@@ -158,9 +158,9 @@ async fn run(ctx: &mut DalContext) {
         .await
         .expect("could not commit and update snapshot to visibility");
 
-    assert!(Action::run(ctx, action.id())
+    assert!(ActionPrototype::run(ctx, proto.id(), component.id())
```

Comment:
What happens when you run `ActionPrototype::run` and no Action was created previously for that component? I don't see we taking that into account in this run code, but am unsure if that's something we can ignore

Reply:
This test is failing lint because the action created above is not being used, but it looks to me like we could remove that statement altogether

Reply:
Nothing should happen, I think we can remove that statement, running the prototype is now the interface for running an action func directly (and all of the graph mutations as a result of that run are processed elsewhere)